archiver.py

```diff
@@ -177,6 +181,30 @@ def progress_cb(bytes_downloaded, total_size):
         sftp.close()
         client.close()
 
+        # We now go and delete any files that are older than the retention
+        # period, if specified
+        if service.retention_period_days:
+            service_base_dir = os.path.join(self.base_dir, service.name)
```

Comment:
erm; it feels odd that we are handling all hosts here whereas a service is host-specific. (What if I wanted to keep logs from hera for 6 months but juno for 1 week?).

And yes, I know that will leave the stuff from old hosts hanging around. Just go and clear them out manually.

Reply:
Oh, good catch

Reply:
Though I'm not sure I want to change the config, as it doesn't currently have any host specific options